app/routes.tsx

```diff
@@ -375,6 +380,22 @@ export const routes = createRoutesFromElements(
                     handle={titleCrumb('New Router')}
                   />
                 </Route>
+                <Route
+                  path="internet-gateways"
+                  handle={{ crumb: 'Internet Gateways' }}
+                  loader={VpcInternetGatewaysTab.loader}
+                  element={<VpcInternetGatewaysTab />}
+                >
+                  <Route
+                    path=":gateway"
+                    element={<EditInternetGatewayForm />}
+                    loader={EditInternetGatewayForm.loader}
+                    handle={makeCrumb(
+                      (p) => p.gateway!,
+                      (p) => pb.vpcInternetGateway(getInternetGatewaySelector(p))
+                    )}
+                  />
```

Comment:
I think this should be a title-only crumb. I tried to follow a rule that side modals don't get a crumb. It looks weird for it to pop up in the background.